src/components/02_atoms/Widgets/TimestampDatetime.js

```diff
@@ -14,6 +14,7 @@ const TimestampDatetime = ({ fieldName, label, value, onChange }) => (
     InputLabelProps={{
       shrink: true,
     }}
+    required={required}
```

Comment:
We should add the newly added prop to `WidgetPropTypes` so that it gets validated.